content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -48,3 +50,25 @@ Be sure to copy the metadata URL (from the final step of configuring Okta) befor
 6. Your Okta SSO configuration is complete and ready to [use](../single-sign-on.html#using-sso).
 
     ![sso-settings](../images/sso/settings-okta.png)
+
+## Configuration - Team Mapping (Okta)
+1. Complete all steps outlined in the [Configuration (Okta)](./okta.html#configuration-okta-) section above, and take note of the default team mapping behavior as described [here](../single-sign-on.html#managing-team-membership-through-sso).
+
+2. Edit your Terraform Cloud Okta Application and complete the following steps:
+* Expand the `Attributes` section of the Application configuration (under the `Sign On` tab):
```

Comment:
The combination of numbered and bulleted lists here is kind of disorganized. I suggest turning the things that are currently numbered list items into normal paragraphs, and turning the things that are currently un-numbered bullet list items into numbered list items. 

The reason I'm suggesting this is because the "1" and "2" items are just explaining the prerequisites, whereas the two bullet points are the actual steps you perform.

Reply:
Agreed.  Updated.